core/src/mindustry/ui/dialogs/ModsDialog.java

```diff
@@ -217,20 +218,19 @@ void setup(){
                                 }).size(50f);
                             }
                         }).growX().right().padRight(-8f).padTop(-8f);
-
-
                     }, Styles.clearPartialt, () -> showMod(mod)).size(w, h).growX().pad(4f);
                     table.row();
-                }
-            });
+                }).margin(10f).top();
 
+                if(mobile && !Core.graphics.isPortrait()){
+                    //hide search bar on mobile, takes up too much space
+                    cont.getCells().get(2).get().clear();
```

Comment:
This is very hacky and will break without an obvious cause if any relevant layout is changed.

Reply:
ok ill add a cons for search bar table